src/dialog-user/services/dialogData.ts

```diff
@@ -43,6 +43,14 @@ export default class DialogDataService {
       }
     }
 
+    if (field.type === 'DialogFieldDateControl') {
+      if (_.isNull(field.default_value) || _.isUndefined(field.default_value)) {
+        field.dateField = field.timeField = new Date();
+      } else {
+        field.dateField = field.timeField = new Date(data.default_value);
```

Comment:
`timeField` both here and on line 48 I don't think mean anything for a `DialogFieldDateControl`, only if the type is a `DialogFieldDateTimeControl`.